mm2src/mm2_event_stream/src/event.rs

```diff
@@ -0,0 +1,47 @@
+use serde_json::Value as Json;
+
+// Note `Event` shouldn't be `Clone`able, but rather Arc/Rc wrapped and then shared.
+// This is only for testing.
+/// Multi-purpose/generic event type that can easily be used over the event streaming
+#[cfg_attr(any(test, target_arch = "wasm32"), derive(Clone, Debug, PartialEq))]
+#[derive(Default)]
+pub struct Event {
+    /// The type of the event (balance, network, swap, etc...).
+    event_type: String,
```

Comment:
The event itself is `Arc`ed (since the json message inside it might be arbitrary large), so we don't really need to wrap the `event_type` here in an `Arc`.
`streamer_id` thought could be `Arc`ed in some places but i went for keeping it simple since it's cloned as many as the events we spit out and not as many as the events we hand to clients. i think we can have this reconsidered though.

Reply:
Alright thanks. Streamer_id def needs to be made into an Arc.. I counted the clones and it much